bzt/jmx/tools.py

```diff
@@ -678,8 +678,15 @@ def __gen_data_sources(self, scenario):
                 if not delimiter:
                     delimiter = guess_delimiter(source_path)
 
-            config = JMX._get_csv_config(source_path, delimiter, source.get("quoted", False), source.get("loop", True),
-                                         source.get("variable-names", ""))
+            random_parameter = source.get("random-order")
+            if not random_parameter:
+                config = JMX._get_csv_config(source_path, delimiter, source.get("quoted", False),
+                                             source.get("loop", True), source.get("variable-names", ""))
```

Comment:
let's change params order and move 'quoted' to the end. It can break other projects (suddenly ;) but makes csv generators more uniform.